ftw/sliderblock/browser/resources/sliderblock.js

```diff
@@ -32,7 +32,7 @@
           var settings = sliderPane.parent().data("settings");
```

Comment:
Does this work in Plone 4 and 5? Shouldn't we use the pattern described in the docu: https://devdocs.4teamwork.ch/plone/javascript/ftw-resource-bundling-plone5/#transform-our-javascript-in-the-product ?
Not sure..

Reply:
I discussed it further with buchi and made changed the code accordingly